src/theme.ts

```diff
@@ -46,10 +46,13 @@ export const theme = {
   },
   borderRadii: {
     md: "7px",
-    md_small: "4px",
-    b: "none",
+    sm: "4px",
+    border: "none",
```

Comment:
I don't know if having an alias just for `border: none` makes sense. If there is a real set of circumstances where we are using this class, maybe name it that?

Also I think it needs to go on a part of theme that matches a CSS property, like `theme.borders.XYZ`.

If this is just an alias with none with no real meaning, I think I'd prefer to just have `none` in the code down the line.